### PR TITLE
add project file structure

### PR DESCRIPTION
**Pull request by amarquesmoura**
This pull request add initial file structure to our project.

### CODE REVIEW

File: js/components/Search/index.js [hunk 1]
```diff
@@ -0,0 +1,5 @@
+import Search from "./Search";
+import SearchContainer from "./SearchContainer";
```
Comment (siddharthparmar7) [inline js/components/Search/index.js:2]: Components do not have containers. Please make changes to your imports and exports.

File: js/components/Search/index.js [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+export { Search };
```
Comment (siddharthparmar7) [inline js/components/Search/index.js:4]: Add missing Search.js file.

File: js/components/MapView/index.js [hunk 3]
```diff
@@ -0,0 +1,5 @@
+import MapView from "./MapView";
+import MapViewContainer from "./MapViewContainer";
```
Comment (siddharthparmar7) [inline js/components/MapView/index.js]: Components do not have container files. Please remove MapViewContainer file. If it's a screen then move it to the screens dir.

File: js/screens/About/index.js [hunk 4]
```diff
@@ -0,0 +1,3 @@
+import About from "./About";
```
Comment (siddharthparmar7) [inline js/screens/About/index.js:1]: About screen is missing a container file. Please add it.

File: js/screens/Contact/index.js [hunk 5]
```diff
@@ -0,0 +1,3 @@
+import Contact from "./Contact";
```
Comment (siddharthparmar7) [inline js/screens/Contact/index.js:1]: Contact Screen is missing a container file. Please add it.

File: js/screens/HowTo/index.js [hunk 6]
```diff
@@ -0,0 +1,3 @@
+import HowTo from "./HowTo";
```
Comment (siddharthparmar7) [inline js/screens/HowTo/index.js]: Make this screen name more descriptive and add a container file.

File: js/screens/Onboarding/index.js [hunk 7]
```diff
@@ -0,0 +1,3 @@
+import Onboarding from "./Onboarding";
```
Comment (siddharthparmar7) [inline js/screens/Onboarding/index.js:1]: Add a container file.

File: js/screens/Resources/index.js [hunk 8]
```diff
@@ -0,0 +1,3 @@
+import Resources from "./Resources";
```
Comment (siddharthparmar7) [inline js/screens/Resources/index.js:1]: Add a container file. Thank you.

File: js/screens/Spot/index.js [hunk 9]
```diff
@@ -0,0 +1,3 @@
+import Spot from "./Spot";
```
Comment (siddharthparmar7) [inline js/screens/Spot/index.js:1]: Add container file. Thank you.

**Review by siddharthparmar7 — CHANGES_REQUESTED** [review]
Please make the changes. Thank you.

**Comment by amarquesmoura** [comment]
Adjustments made as requested.

**Comment by siddharthparmar7** [comment]
Please rebase your develop branch and then update your feature branch. To get the latest project structure on your machine.

**Comment by amarquesmoura** [comment]
Done.

**Review by siddharthparmar7 — APPROVED** [review]

**Pull request merged**